bin/.eslintrc

```diff
@@ -0,0 +1,14 @@
+{
+  "env": {
+    "node": true,
+    "commonjs": true,
+    "es6": false
+  },
+  "rules": {
```

Comment:
eslint indent rules are duplicating what prettier does.
See https://github.com/prettier/prettier#eslint for details how to integrate both these tools at the same time.